Week2/homework/app.js

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env node
+'use strict';
+
+const util = require('util');
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyf_Pass1',
+  database: 'new_world',
+});
+
+const execQuery = util.promisify(connection.query.bind(connection));
+
+const command = process.argv[2];
+const query = process.argv[3];
+
+connection.connect();
+
+try {
+  switch (command) {
+    case 'capital':
+      const capitalOfCountry = `SELECT c.Name,ci.Name FROM country AS c 
+                                LEFT JOIN city AS ci
+                                ON c.Capital=ci.ID
+                                WHERE c.Name="${query}";`;
```

Comment:
Can you update the queries to use prepared statements? https://github.com/mysqljs/mysql